LPS-168734

fortunatomaldonado opened the pull request:
https://issues.liferay.com/browse/LPS-168734

Hello Selena, I am having difficulties testing this component on my end and was wondering if you had time to test this.
I am not sure if it resolves the issue but I believe it might.

Please let me know if there are any questions or comments about this.
Thank you!

Commit:
LPS-162822 Create method to validade next Object State

natocesarrego committed:
LPS-163638 Add new configuration related to locale in friendly URLs

natocesarrego committed:
LPS-163638 Add language keys

natocesarrego committed:
LPS-163638 buildLang

natocesarrego committed:
LPS-163638 Use as the default value the value of property locale.prepend.friendly.url.style

alessiorendina committed:
COMMERCE-5205 Added commerce inventory method interface

alessiorendina committed:
COMMERCE-5205 Added commerce inventory method registry

alessiorendina committed:
COMMERCE-5205 Added commerce inventory method comparator

alessiorendina committed:
COMMERCE-5205 Added commerce inventory engine contributor interface

alessiorendina committed:
COMMERCE-5205 Added commerce inventory engine contributor comparator

alessiorendina committed:
COMMERCE-5205 Added commerce inventory engine contributor registry

alessiorendina committed:
COMMERCE-5205 Added default commerce inventory method implementation

alessiorendina committed:
COMMERCE-5205 Added new configuration

alessiorendina committed:
COMMERCE-5205 Moved configuration in another package

alessiorendina committed:
COMMERCE-5205 Rename constant

alessiorendina committed:
COMMERCE-5205 Refactor commerce inventory engine

alessiorendina committed:
COMMERCE-5205 Export packages

alessiorendina committed:
COMMERCE-5205 Added required dependencies

alessiorendina committed:
COMMERCE-5205 Added new localized key

alessiorendina committed:
COMMERCE-5205 Semantic versioning (gw baseline)

alessiorendina committed:
COMMERCE-5205 Refactor display context

alessiorendina committed:
COMMERCE-5205 Added parameter to match constructors

alessiorendina committed:
COMMERCE-5205 Refactor edit catalog screen and action command

natocesarrego committed:
LPS-163638 Get configuration from preferences instead of property

alessiorendina committed:
COMMERCE-5205 Update usages

natocesarrego committed:
LPS-163638 Set Company in ThemeDisplay to fix Integration Tests

MarinhoFeliphe committed:
LPS-162822 Inline, rename and keep it simple

Commit:
COMMERCE-5205 contributors must be in the method not in the engine

ambrinchaudhary committed:
LPS-167635 Add smae validation than when creating an alias

ambrinchaudhary committed:
LPS-167635 Empty url should not show an error

ambrinchaudhary committed:
LPS-167635 Error message should fit in its container

natocesarrego committed:
LPS-163638 Use preferences instead of property to fix Integration Tests

AliciaGarciaGarcia committed:
LPS-167635 allow relative URLs

Commit:
COMMERCE-5205 Build lang

Commit:
LPS-167508 Move ObjectEntryEntityModel back to object-rest-impl as is an implementation and for consistency with the rest of EntityModels

Commit:
LPS-167508 Baseline

ambrinchaudhary committed:
LPS-167635 Move form to react component

ambrinchaudhary committed:
LPS-167635 Improve help message

ambrinchaudhary committed:
LPS-167635 buildLang

ambrinchaudhary committed:
LPS-168310 Disable save button whenever there is a pattern with error

pedro-oliveira446 committed:
LPS-166681 displaying new fields in form edit and removing readonly property

Commit:
COMMERCE-5205 no need for ConfigurationBeanDeclaration after LPS-162830

ambrinchaudhary committed:
LPS-168310 Remove error if the row is deleted

ambrinchaudhary committed:
LPS-168310 SF

Commit:
LPS-167508 Adapt test to get the entity fields from the object definition and ensure that it is creating the expected entity fields

MrJohn1911 committed:
LPS-168709 Remove immediate from commerce service module

MrJohn1911 committed:
LPS-168709 Remove immediate from commerce ship module

MrJohn1911 committed:
LPS-168709 Remove immediate from headless commerce module

torresvictor100 committed:
LPS-163648 Create JSP for default_portlet_decorator

ambrinchaudhary committed:
LPS-168310 This method could be private

ambrinchaudhary committed:
LPS-168310 Avoid using separate list. Each pattern will know if it has error or not

ambrinchaudhary committed:
LPS-168310 constant function

ambrinchaudhary committed:
LPS-168310 Remove unneccesary jsp

torresvictor100 committed:
LPS-163648 Add Language Key

natocesarrego committed:
LPS-163638 Omit default value to simplify

torresvictor100 committed:
LPS-163648 gw buildLang

carolmariaabb committed:
LPS-166681 Simplify code

torresvictor100 committed:
LPS-163648 change the PropsKeys

ealonso committed:
LPS-169700 Auto

ealonso committed:
LPS-169700 Uses LayoutUtilityPageEntryTypesConstants

ealonso committed:
LPS-169700 No needed

ealonso committed:
LPS-169700 baseline

p2kmgcl committed:
LPS-168309 Remove unneeded taglib

rafaprax committed:
LPS-166694 Remove immediate from portal-search-elasticsearch7-impl

rafaprax committed:
LPS-166694 Remove immediate from portal-search-admin-web

rafaprax committed:
LPS-166694 Remove immediate from portal-search-rest-impl

rafaprax committed:
LPS-166694 Remove immediate from portal-search-web

rafaprax committed:
LPS-166694 Remove immediate from portal-search

rafaprax committed:
LPS-166694 Remove immediate from portal-search-elasticsearch-cross-cluster-replication-impl

rafaprax committed:
LPS-166694 Remove immediate from portal-search-elasticsearch-monitoring-web

rafaprax committed:
LPS-166694 Remove immediate from portal-search-learning-to-rank-impl

rafaprax committed:
LPS-166694 Remove immediate from portal-search-similar-results-web

rafaprax committed:
LPS-166694 Remove immediate from portal-search-tuning-rankings-web

rafaprax committed:
LPS-166694 Remove immediate from portal-search-tuning-synonyms-web

rafaprax committed:
LPS-166694 Remove immediate from portal-search-tuning-web

rafaprax committed:
LPS-166694 Remove immediate from search-experiences-service

rafaprax committed:
LPS-166694 Remove immediate from search-experiences-web

veroglez committed:
LPS-167055 Improve accessibility for the product menu

adolfopa committed:
LPS-169174 Move logic into the display context

adolfopa committed:
LPS-169174 Rename methods

adolfopa committed:
LPS-169174 Inline method

adolfopa committed:
LPS-169174 Auto SF

adolfopa committed:
LPS-169174 Create management toolbar display context

adolfopa committed:
LPS-169174 Move logic to mgmt toolbar display context

adolfopa committed:
LPS-169174 Inline JSP

adolfopa committed:
LPS-169174 Move logic into display context

rafaprax committed:
LPS-166694 Remove immediate from portal-search-test

MrJohn1911 committed:
LPS-168709 Removed service property from Commerce Message Configurator and let it be implied as immediate

nNilton committed:
LPS-169198 add type validation

nNilton committed:
LPS-169198 sort

murilostodolni committed:
LPS-162822 Rename

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

ealonso committed:
LPS-168309 Removes useless <liferay-frontend:fieldset-group> taglib usages in all portal. It is adding a <div aria-multiselectable="true"> to all forms, and it should be added to select

ealonso committed:
LPS-168309 Removes useless taglib

ealonso committed:
LPS-168309 Replace useless <aui:fieldset-group> taglib usage with <div class="sheet"><div class="panel-group panel-group-flush"> in all portal. It is adding <div aria-multiselectable="true" role="tablist"> which is not necessary

ealonso committed:
LPS-168309 Removes useless taglib

ealonso committed:
LPS-168309 Breaking changes

ealonso committed:
LPS-168309 baseline

adolfopa committed:
LPS-169174 Use dropdowns instead of menu items

adolfopa committed:
LPS-169174 Change API to use dropdown items

murilostodolni committed:
LPS-162822 SF

adolfopa committed:
LPS-169174 Rename classes and methods to match purpose

adolfopa committed:
LPS-169174 Sort arguments

adolfopa committed:
LPS-169174 Baseline

rafaprax committed:
LPS-166694 Let component be implied as immediate

pei-jung committed:
LPS-167330 (user-groups-admin-web) - add aria label to the add button

SelenaAungst commented:
Hi @fortunatomaldonado,

I tested this and was unfortunately unable to get any changes to show up as well. I tried applying your changes in some other files that looked potentially relevant too (translation/translation-web/.../TranslateFieldSetEntries.js, template/template-web/.../Field.js, layout/layout-page-template-admin-web/.../FormField.es.js) but had no luck.

Please let me know if there is anything I can do to help with this!

pei-jung committed:
LPS-167330 (frontend-taglib-clay) - add aria label to the management toolbar's search and view opitons button

smottal committed:
COMMERCE-9405 Add catalog view permission to Discount Manager

boton committed:
LPS-168310 SF

ambrinchaudhary committed:
LPS-168310 Fix SF error

claraizquierdo committed:
LPS-167441 Add more context to aria-labels for String inputs in the criteria builder

titokenzo committed:
LPS-169459 Update number validation for hidden input

pei-jung committed:
LPS-169455 Use ServiceContextFactory to create the service context to ensure attributes are being set properly

adolfopa committed:
LPS-169174 Use correct mvc command name (default)

adolfopa committed:
LPS-169522 Provide default value when reading sorting order

Otherwise, the management toolbar won't display the sorting button.

adolfopa committed:
LPS-169522 Add attribute to customize file entry permission checking

adolfopa committed:
LPS-169522 Inject display context in mvc command

adolfopa committed:
LPS-169522 Remove unnecessary constant

adolfopa committed:
LPS-169522 Delegate update/delete permission checks into portlet

This replicates the behavior of the old blog images browser, where if
a user had permission to add an entry, then mutation of the repository
was allowed.

Commit:
LRAC-12446 Add the macro to wait for the events to be sent to the forms visualization

gamerson committed:
LPS-169543 add support for properties field in client-extension.yaml mapping

markocikos committed:
LPS-169541 Add samples of request success and errors

ambrinchaudhary committed:
LPS-168310 Use vars for the error message examples

ambrinchaudhary committed:
LPS-168310 buildLang

markocikos committed:
LPS-169541 Fix dropdown item async action error handling

markocikos committed:
LPS-169541 Remove unused

BarbaraCabrera committed:
LPS-168654 Replace liferay-ui:icon-menu with clay:dropdown-menu in scope.jsp

marcelabc committed:
LPS-166551 Remove FF of update the current object entry action

markocikos committed:
LPS-169541 Equalize row height when loading indicator is present

devpedrotavares committed:
LPS-169630 include validations before dataDefinition deletion

brianchandotcom committed:
LPS-167508 SF

gamerson committed:
LPS-168901 forgot to readd themeFavicon

austinchiang committed:
LPS-169174 Use more specific xpaths

jaimelr10 committed:
LPS-163941 Deprecate robotType in favour of Type for macro addImageFromUrl

jaimelr10 committed:
LPS-163941 Add macros to addImage and viewImageIsPresent

jaimelr10 committed:
LPS-163941 Create test to check that an image can be uploaded from the Wiki image uploader

smottal committed:
COMMERCE-9405 Semantic versioning

ealonso committed:
LPS-169700 Upgrade process

adolfopa committed:
LPS-169522 Semver

ealonso committed:
LPS-169700 Includes new utility page on rest api

ealonso committed:
LPS-169700 buildRest

ealonso committed:
LPS-169700 Convert type from int to String

silasbsi committed:
LPS-168018 Colored bubble of the applications status does not appear

claraizquierdo committed:
LPS-167441 BuildLang

veroglez committed:
LPS-167055 BuildLang

Commit:
LPS-168710 Do not put 'model' key in variables, when the payloadJSONObject doesn't have it

lfbesada committed:
LPS-169664 Modify the upgrade process in order to set the proper scopeType instead of clearing the article selection

lfbesada committed:
LPS-169664 Adapt the portletSetupTitle if apply

ealonso committed:
LPS-169729 SF

ealonso committed:
LPS-169729 Uses the same editor for web content display and for web content mapping

baharturk committed:
COMMERCE-10370 Fix failing tests using macro checkSuccessAlertAppearsAndDisappears

lfbesada committed:
LPS-169664 Set the proper scopeLayoutUuid if apply

lorenzolapescara committed:
COMMERCE-9405 Add test

interaminense committed:
LPS-153738 Forms should be able to accept autofill input (browser extensions)

daiazius committed:
LPS-167659 Create a mapper for DuplicateListTypeDefinitionExternalReferenceCodeException and remove unnecessary validation

daiazius committed:
LPS-167659 Delete unused exception and mapper

carolmariaabb committed:
LPS-168449 Sort

carolmariaabb committed:
LPS-168449 Add support for multipicklist object fields in the page editor

carolmariaabb committed:
LPS-168449 The value may not come with the bracket symbol, as in the case of sending the entry in the page editor

Commit:
LPS-169207 Fixed the Drivers Column is Misspelled

rafaellajordao committed:
LPS-168702 refactor DEBooleanField#CanBeSubmittedAndCheckEntries

rafaellajordao committed:
LPS-168703 refactor COLOR_SWATCH_ITEM path

lfbesada committed:
LPS-169664 Add an integration test to ensure that when a Global scoped Journal Article is selected, its scoped-related configuration is adapted if apply

brianchandotcom committed:
artifact:ignore commerce-product-content-web 4.0.75 prep next

brianchandotcom committed:
artifact:ignore oauth2-provider-service 4.0.43 prep next

brianchandotcom committed:
artifact:ignore headless-commerce-admin-account-impl 4.0.54 prep next

brianchandotcom committed:
LPS-168310 Match destinationURL in that same Java class

lfbesada committed:
LPS-169664 Add an assert to assert that nothing changes when the scoped-related configuration match with the scoped article

lfbesada committed:
LPS-169664 Add tests to ensure the behavior when is selected a site-scoped or a layout-scoped Journal Article

lfbesada committed:
LPS-169664 Add a test to ensure the scoped preferences are not updated when no article is selected

brianchandotcom committed:
LPS-163638 Minor SF

Commit:
LPS-167508 Avoid ambiguity

ling-alan-huang committed:
LPS-167788 Auto SF

liferay-translation committed:
LPS-77699 Update Translations

Commit:
LPS-169177 Refactor locator

Commit:
LPS-169177 Modify case to assert subtitle

Commit:
LPS-169762 Modify locators to adapt new behavior

qz1010 committed:
LPS-169488 SF boolean type name

qz1010 committed:
LPS-169488 Auto SF

Hijky committed:
LPS-153326 Add a macro to get and delete the relationship with the system object

Hijky committed:
LPS-153326 Add a macro to set global object entry id

Hijky committed:
LPS-153326 Update a testcase to modify the method of deleting user accounts

Hijky committed:
LPS-153326 Add a testcase to make a relationship with a system object through API

slnn committed:
LPS-169735 Revert "LPS-169259 Remove "immediate = true" from module portal-settings"

This reverts commit acda134e7dc2b87162707d6464f7ccc812f77020.

jkappler committed:
Revert "LPS-168998 Remove no needed code"

This reverts commit be20c688233c7447a628c1bebdcf906e71f4f95b.

lfbesada committed:
LPS-169291 Use data-lfr-drop-zone-id element attribute as id

lfbesada committed:
LPS-169291 Prevent from consider as a null id when the attribute value is "null"

lfbesada committed:
LPS-169291 SF rename vars

JoyceWang08211 committed:
LRQA-78653 Add related tests to the relevant suite for portal-settings module

austinchiang committed:
LPS-163941 Remove unnecessary step

jkappler committed:
LPS-169798 New finder to retrieve a page template collection by name

jkappler committed:
LPS-169798 Baseline

baharturk committed:
COMMERCE-10370 Fix another failing test using macro checkSuccessAlertAppearsAndDisappears

jkappler committed:
LPS-169798 Fallback to the name if we cannot find by key

jkappler committed:
LPS-169798 Control at the end if the collection is null or not

austinchiang committed:
LPS-169174 Fix more tests

jkappler committed:
LPS-169798 Adds integration tests. No new method was created for asserting layoutsImporterResultEntries since it's specific logic for the test

ealonso committed:
LPS-168654 Typo

ealonso committed:
LPS-168654 Adds necessary logic to allow selecting scopes

liferay-translation committed:
LPS-77699 Update Translations

Commit:
LPS-169776 Change so that instead of vertical button alignment we have a horizontal button alignment

jkappler committed:
LPS-169776 Use correct label

Commit:
LPS-169815 portal-search-web: Fix Category Facet NP

Hanlf committed:
LPS-169793 Update locators for the new UI

claraizquierdo committed:
LPS-167441 Update tests and snapshots

Commit:
LPS-169825 Add Translate For Commerce Web Content

brianchandotcom committed:
LPS-167508 Rename

jkappler committed:
LPS-169811 We cannot use specific file entries in the xml markup, so create the file entry first and substitute it

ambrinchaudhary committed:
LPS-167099 Add aria-label and title to the history icon

jkappler committed:
LPS-169811 Ignore the test for now, since it was created to fix performance issues. We'll unignore once the performance has improved, otherwise it will break the CI

brianchandotcom committed:
LPS-167508 Can you change all of ERCFieldName to ERCObjectFieldName (we may need an upgrade because of ObjectFieldSettingConstants.java

jkappler committed:
LPS-169811 Apply also for JournalExportImportTest

jkappler committed:
LPS-169811 Ignore the test for now, since it was created to fix performance issues. We'll unignore once the performance has improved, otherwise it will break the CI

danvb112 committed:
LPS-169828 Add FF validation when submitting an action

brianchandotcom committed:
LPS-169664 Use interface

brianchandotcom committed:
LPS-169664 Sort

brianchandotcom committed:
LPS-169664 SF

brianchandotcom committed:
LPS-169664 SF

brianchandotcom committed:
LPS-169664 SF

Commit:
LPS-169825 Fix Account Subscriptions Status

brianchandotcom committed:
LPS-168449 SF

brianchandotcom committed:
LPS-167441 Wordsmith

brianchandotcom committed:
LPS-167441 Regen

brianchandotcom committed:
LPS-169488 Sort

brianchandotcom committed:
LPS-169488 rename attribute

brianchandotcom committed:
LPS-169630 Rename

brianchandotcom committed:
LPS-169630 Sort by exception name

brianchandotcom committed:
LPS-169776 SF

brianchandotcom committed:
LPS-153738 can I sort this?

brianchandotcom committed:
LPS-167055 SF

brianchandotcom committed:
LPS-169543 Wrong name

brianchandotcom committed:
LPS-169543 As used

brianchandotcom committed:
LPS-169543 Rename to match usage in modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/internal/client/extension/NodeBuildConfigurer.java

brianchandotcom committed:
LPS-169543 prep next

brianchandotcom committed:
LPS-168654 SF

brianchandotcom committed:
COMMERCE-9405 Not needed

brianchandotcom committed:
LPS-169798 SF

CalumR23 committed:
POSHI-444 Generate CHANGELOG

CalumR23 committed:
POSHI-444 Update default poshi runner version

CalumR23 committed:
POSHI-444 Update poshi-core version

brianchandotcom committed:
COMMERCE-5205 I ran 'ant update-portal-osgi-configuration-properties'

ealonso committed:
LPS-169811 Moves test to JournalArticleLocalServiceTest and check JournalArticleDisplay.getContent(), since it is the logic that we are executing on web content display portlet to show the content of a web content

shuyangzhou committed:
LPS-169811 Add logging timer

brianchandotcom committed:
POSHI-444 prep next

brianchandotcom committed:
POSHI-444 prep next

brianchandotcom committed:
POSHI-444 prep next

shuyangzhou committed:
LPS-169811 Remove stream usage

brianchandotcom committed:
LPS-168309 #sfme please enforce

brianchandotcom committed:
LPS-169455 SF

brianchandotcom committed:
LPS-162822 Rename

fortunatomaldonado committed:
LPS-168734 Add role=alert to announce required fields when press enter

fortunatomaldonado commented:
Thank you for your help!
It turns out yarn was caching the file and not updating it. The following helped change it.
```
yarn build --clean
gw deploy
```
